cfme/services/myservice.py

```diff
@@ -66,11 +66,21 @@ class EditTagsForm(MyServicesView):
 class MyServiceDetailView(MyServicesView):
     title = Text("#explorer_title_text")
 
-    properties_detail = SummaryTable(title='Properties')
-    lifecycle_detail = SummaryTable(title='Lifecycle')
-    relationships_detail = SummaryTable(title='Relationships')
-    vm_detail = SummaryTable(title='Totals for Service VMs ')
-    smart_mgmt_detail = SummaryTable(title='Smart Management')
+    @View.nested
+    class details(Tab):  # noqa
+        properties_detail = SummaryTable(title='Properties')
```

Comment:
**optional**
the naming here looks gone bad now that the "detail" is part of structure - we should at least follow up with a cleanup of the name